doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc

```diff
@@ -15,8 +15,12 @@ endif::DM[]
 .Procedure
 . If the *Installation* or *Security* tab is open, click *Next* until you view the *Console* tab.
 . If you created the secret for {CENTRAL} according to the instructions in <<secrets-central-create-proc>>, enter the name of the secret in the *Secret* field.
+. In an authoring environment, you can use Git hooks to facilitate interaction between the internal Git repository of {CENTRAL} and an external Git repository. If you want to use Git hooks, you must prepare a Git hooks directory in an OpenShift configuration map, secret, or persistent volume claim object in the project namespace. To use this directory, make the following changes:
+.. Under *GitHooks*, in the *Mount path* field, enter a fully qualified path for the directory, for example, `/opt/kie/data/git/hooks`.
```

Comment:
Minor suggested rewrite:

Under *GitHooks*, in the *Mount path* field, enter a fully qualified path for the directory. For example, `/opt/kie/data/git/hooks`.